var/spack/repos/builtin/packages/py-warpx/package.py

```diff
@@ -38,4 +38,4 @@ class PyWarpx(PythonPackage):
     depends_on('warpx +lib +mpi +shared', type=('build', 'link'), when='+mpi')
 
     def setup_build_environment(self, env):
-        env.set('PYWARPX_LIB_DIR', self.spec['warpx'].prefix.lib)
+        env.set('PYWARPX_LIB_DIR', self.spec['warpx'].libs.directories[0])
```

Comment:
Shouldn't `.prefix.lib` point to the direct directory @adamjstewart ? I want to point to `<prefix>/lib[64]/` here.

Reply:
`.prefix.lib` points to `<prefix>/lib`, `.libs.directories[0]` points to `<prefix>/lib[64]`

Reply:
Ouch. I feel like I want to add `-DCMAKE_INSTALL_LIBDIR=lib` to the `warpx` package just so I do not have to remember  this hack :smiley:

Reply:
Done :)